docs/static/examples/FlipCard.js

```diff
@@ -0,0 +1,160 @@
+    flex: 1,
+    backgroundColor: '#baeee5',
+    borderRadius: 16,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+});
+
+const FlipCard = ({
+                    isFlipped,
+                    cardStyle,
+                    direction = 'y',
+                    duration = 500,
+                    RegularContent,
+                    FlippedContent,
+                  }) => {
+  const isDirectionX = direction === 'x';
+
+  const regularCardAnimatedStyle = useAnimatedStyle(() => {
+    const spinValue = interpolate(Number(isFlipped.value), [0, 1], [0, 180]);
+    const rotateValue = withTiming(`${spinValue}deg`, { duration });
+
+    return {
+      transform: [
+        isDirectionX ? { rotateX: rotateValue } : { rotateY: rotateValue },
+      ],
+    };
+  });
+
+  const flippedCardAnimatedStyle = useAnimatedStyle(() => {
+    const spinValue = interpolate(Number(isFlipped.value), [0, 1], [180, 360]);
+    const rotateValue = withTiming(`${spinValue}deg`, { duration });
+
+    return {
+      transform: [
+        isDirectionX ? { rotateX: rotateValue } : { rotateY: rotateValue },
+      ],
+    };
+  });
+
+  return (
+      <View>
+        <Animated.View
+            style={[
+              flipCardStyles.regularCard,
+              cardStyle,
+              regularCardAnimatedStyle,
+            ]}>
+          {RegularContent}
+        </Animated.View>
+        <Animated.View
+            style={[
+              flipCardStyles.flippedCard,
+              cardStyle,
+              flippedCardAnimatedStyle,
+            ]}>
+          {FlippedContent}
+        </Animated.View>
+      </View>
+  );
+};
+
+const flipCardStyles = StyleSheet.create({
+  regularCard: {
+    position: 'absolute',
+    zIndex: 1,
+  },
+  flippedCard: {
+    backfaceVisibility: 'hidden',
+    zIndex: 2,
+  },
+});
+
+export default function App() {
+  const isFlipped = useSharedValue(false);
+
+  const handlePress = () => {
+    isFlipped.value = !isFlipped.value;
+  };
+
+  return (
+      <SafeAreaView style={styles.container}>
+        <FlipCard
+            isFlipped={isFlipped}
+            cardStyle={styles.flipCard}
+            FlippedContent={<FlippedContent />}
+            RegularContent={<RegularContent />}
+        />
+        <View style={styles.buttonContainer}>
+          <Pressable style={styles.toggleButton} onPress={handlePress}>
+            <Text style={styles.toggleButtonText}>Toggle card</Text>
+          </Pressable>
+        </View>
+      </SafeAreaView>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    height: 300,
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  buttonContainer: {
+    marginTop: 16,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  toggleButton: {
+    backgroundColor: '#b58df1',
+    padding: 12,
+    borderRadius: 48,
+  },
+  toggleButtonText: {
+    color: '#fff',
+    textAlign: 'center',
+  },
+  flipCard: {
+    width: 150,
```

Comment:
```suggestion
    width: 170,
```
i would make it a little but wider because now it feels crowded